architecture/additional_concepts/admission_controllers.adoc

```diff
@@ -22,6 +22,18 @@ system configured defaults. In addition, admission control plug-ins may modify
 related resources as part of request processing to do things such as
 incrementing quota usage.
 
+[WARNING]
+====
+The {product-title} master has a default list of plug-ins that are enabled by
+default for each type of resource (Kubernetes and {product-title}). These are
+required for the proper functioning of the master. It is not recommended to
```

Comment:
"It is not recommended..." is expletive. Maybe:

"Modifying these lists is not recommended..."